### ISSUE TITLE
Vehicle marker scaling

### ISSUE THREAD

**Issue by roata** [roataway/web-ui#50]
Our vehicle markers are always of the same size, regardless of the zoom level.

High-level overview:
![image](https://user-images.githubusercontent.com/48152696/66700919-7628b500-ecf6-11e9-89e1-9dedb98eb2f4.png)

Detailed view:
![image](https://user-images.githubusercontent.com/48152696/66700923-86409480-ecf6-11e9-89ec-62913ae41ca8.png)

Markers should be smaller when we're looking at a high-level view, and bigger when we zoom in. At some point, when hundreds of vehicles will be tracked - the map will turn into a "sea of red", which is not informative in any way.

**Comment by iamandrewluca** [comment]
Note:
* https://github.com/Leaflet/Leaflet.markercluster

**Comment by roata** [comment]
Yeah, I guess that would be even better, though I am not sure how it would look like in a dynamic setting. If we're looking at buildings, that's alright, but we're dealing with moving targets, I wonder how the clusters will behave in this case :-)
